cmd/algokey/multisig.go

```diff
@@ -93,3 +105,78 @@ var multisigCmd = &cobra.Command{
 		}
 	},
 }
+
+var convertCmd = &cobra.Command{
+	Use:   "convert -t [transaction file] -p \"[threshold] [PK1] [PK2] ...\"",
+	Short: "Adds the necessary fields to a transaction that is sent from an account to was rekeied to a multisig account.",
```

Comment:
Could you drop the "." from the end of the line ? We don't add "." on other commands.